client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -294,3 +294,20 @@ $width-left-panel-card: 55%;
 	}
 }
 
+.email-provider-stacked-features__logos {
+	column-gap: 3px;
+	display: flex;
+	flex-wrap: wrap;
+	margin-left: 30px;
```

Comment:
We should increase that margin to `32px` so logos are aligned with the text above.